test/OpenTelemetry.AotCompatibility.TestApp/OpenTelemetry.AotCompatibility.TestApp.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>$(TargetFrameworksForAotCompatibilityTests)</TargetFrameworks>
```

Comment:
If you switch this to use `SupportedNetTargetsWithNet9` can we remove `TargetFrameworksForAotCompatibilityTests` from props?

Reply:
Yes. I missed that one. I'll update it.